src/ServiceMonitor/Main.cpp

```diff
@@ -18,9 +18,12 @@ VOID CtrlHandle(DWORD dwCtrlType)
         SetEvent(g_hStopEvent);
         g_hStopEvent = INVALID_HANDLE_VALUE;
         break;
+
     default:
-        return;
+        break;

```

Comment:
Should we return true or false for default?

Reply:
We should return TRUE for default. Otherwise the default handler is supposed to exit the process.